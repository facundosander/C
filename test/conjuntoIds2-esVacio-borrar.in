# crea un conjunto con rango 1..10
crearConjunto 10

# verifica si el conjunto es vacío (lo es)
esVacioConjunto

# inserta 7 elementos
insertarConjunto 1
insertarConjunto 2
insertarConjunto 3
insertarConjunto 5
insertarConjunto 6
insertarConjunto 7
insertarConjunto 9
# imprime el conjunto
imprimirConjunto

# verifica si el conjunto es vacío (no lo es)
esVacioConjunto

borrarConjunto 1
# imprime el conjunto borrando el 1
imprimirConjunto

borrarConjunto 3
borrarConjunto 5
borrarConjunto 7
borrarConjunto 9
# borra los impares e imprime solo elementos pares
imprimirConjunto 

# borra el resto de elementos e imprime el conjunto vacio 
borrarConjunto 2
borrarConjunto 6
imprimirConjunto 

# libera el conjunto 
liberarConjunto  
Fin
