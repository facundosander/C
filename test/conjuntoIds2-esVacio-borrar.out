1>#  crea un conjunto con rango 1..10.
2>3>#  verifica si el conjunto es vacío (lo es).
4>El conjunto de ids está vacío.
5>#  inserta 7 elementos.
6>7>8>9>10>11>12>13>#  imprime el conjunto.
14>1 2 3 5 6 7 9 
15>#  verifica si el conjunto es vacío (no lo es).
16>El conjunto de ids NO está vacío.
17>18>#  imprime el conjunto borrando el 1.
19>2 3 5 6 7 9 
20>21>22>23>24>#  borra los impares e imprime solo elementos pares.
25>2 6 
26>#  borra el resto de elementos e imprime el conjunto vacio .
27>28>29>
30>#  libera el conjunto .
31>32>Fin.
